t/lib/MemcachedTest.pm

```diff
@@ -384,11 +384,15 @@ sub new_sock {
     if ($self->{domainsocket}) {
         return IO::Socket::UNIX->new(Peer => $self->{domainsocket});
     } elsif (MemcachedTest::enabled_tls_testing()) {
-        return eval qq{ IO::Socket::SSL->new(PeerAddr => "$self->{host}:$self->{port}",
-                                    SSL_verify_mode => IO::Socket::SSL::SSL_VERIFY_NONE,
-                                    SSL_cert_file => '$client_crt',
-                                    SSL_key_file => '$client_key');
-                                    };
+        my $ssl_session_cache = shift;
```

Comment:
the eval was here because, if missing, the test suite would blow up with a compile error if you're missing the TLS related perl packages, even if you're not building memcached with `--enable-tls`. You can see how this works in a few other sections that call the ::SSL stuff.

You can still shift in these variables and then use the original return eval; it had $client_key and stuff in it already.